pop-cli-for-appchains/guides/calling-a-chain.md

```diff
@@ -0,0 +1,72 @@
+# Call a chain
+
+### Interactive Guidance (Recommended)
+
+Interact with a chain **using** Pop CLI's interactive guidance by simply entering:
+
+```shell
+pop call chain
+```
+
+You will be prompted to select a pallet, choose a dispatchable function, specify any required arguments, then provide connection details and the information of the account that will sign the extrinsic.
+
+<figure><img src="../.gitbook/assets/callchain.gif" alt="pop call chain"><figcaption><p>pop call chain</p></figcaption></figure>
+
+### Manual (non-interactive)
+
+If you prefer not to use interactive prompts, you can call the chain by specifying all the required arguments directly. For example, to call a dispatchable function on a specific pallet:
+
+```shell
+pop call chain --pallet System --function remark --args "0x11" --url ws://localhost:9944/ --suri //Alice
```

Comment:
This will still prompt for sudo and `skip-confirm` which means it is still interactive

Reply:
https://github.com/r0gue-io/pop-docs/pull/12/commits/0447185c620cce672862cfcdc515396c75e85671